lib/components/viewers/RouteRow.js

```diff
@@ -68,6 +89,31 @@ const SimpleRouteRenderer = styled.span`
   padding: 0;
 `
 
+const PatternButton = styled.button`
+  background: transparent;
+  border: none;
+  border-radius: 50%;
+  color: #004f96;
+  opacity: ${(props) => (props.display ? 100 : 0)};
+  height: 40px;
+  margin-right: 8px;
+  position: relative;
+  transition: all ease-in-out 0.2s;
+  width: 40px;
+  z-index: ${(props) => (props.display ? 2 : -2)};
+
+  svg {
+    margin-top: -3px;
+    height: 22px !important;
+    width: 22px !important;
+  }
+
```

Comment:
There's no focus effect! This lack of feedback is confusing

Reply:
Safari does not handle focus on buttons by default, users adjust their keyboard settings to be able to use keyboard navigation: https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/

Reply:
This makes sense for keyboard navigation, but there is also no styling when I press down on a row item with my mouse!

Reply:
I can add an `:active` state across all browsers! Would we want to add a state for whether the route is selected as well? This would be in addition to the blue outline in other browsers.

Reply:
Absolutely!

Reply:
Okay I overpromised undelivered on this one. After testing a few things, I decided to leave out the `isActive` state, because everything I tried looked very messy on browsers that do have the focus outlines. The `:active` is back though!

Reply:
I think if we provide our own outline style we can remove the browser-built-in ones!